README.rst

```diff
@@ -166,21 +166,25 @@ Export transmart api client observation dataframe to tsv file
     }
 
 
-Patient, diagnosis, biosource and biomaterial export
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+CSR export
+++++++++++
 
-Exports patient > diagnosis > biosource > biomaterial hierarchy as first 4 columns of the file.
-The rest of the columns are concepts. Higher level concepts (e.g Age that is specific to Patient level)
+`The Central Subject Registry (CSR) data model` specific export.
```

Comment:
Missing underscore after the link: should be `` `The Central Subject Registry (CSR) data model`_``